SharedUtils/src/main/java/com/braintreepayments/api/BraintreeSharedPreferences.java

```diff
@@ -27,7 +28,7 @@ private BraintreeSharedPreferences() {}
 
     static SharedPreferences getSharedPreferences(Context context) {
         try {
-            MasterKey masterKey = new MasterKey.Builder(context)
+            MasterKey masterKey = new MasterKey.Builder(context, BRAINTREE_KEY_ALIAS)
```

Comment:
So basically this change switches from using the [default key alias](https://developer.android.com/reference/androidx/security/crypto/MasterKey#DEFAULT_MASTER_KEY_ALIAS()), to using `com.braintreepayments.api.masterkey`.

I know we use `BraintreeSharedPreferences` for config caching, but is there anything else it is used for that is merchant-facing, where changing this key value could break a merchant integration? Could they be relying on it for anything?

Reply:
Not that I can think of. The storage is mostly ephemeral, even config is cached for 5 minutes at a time I believe.